src/project/internal/projectactionscontroller.cpp

```diff
@@ -962,6 +967,36 @@ bool ProjectActionsController::saveProjectToCloud(CloudProjectInfo info, SaveMod
     return ret;
 }
 
+void ProjectActionsController::alsoShareAudioCom(const AudioFile& audio)
+{
+    if (!configuration()->showAlsoShareAudioComDialog()) {
+        shareAudio(audio);
+        return;
+    }
+
+    QUrl audioComUrl = audioComService()->cloudInfo().url;
```

Comment:
You can use cloudInfo from authorization
`audioComService()->authorization()->cloudInfo().url;`